tools/api_proto_plugin/annotations.py

```diff
@@ -5,7 +5,7 @@
 import re
 
 # Key-value annotation regex.
-ANNOTATION_REGEX = re.compile('\[#([\w-]+?):(.*?)\]\s?', re.DOTALL)
+ANNOTATION_REGEX = re.compile('\[#([\w-]+?):(.*?)\](\s?)', re.DOTALL)
```

Comment:
Why not just insert the space, i.e. have it be `[#next-free-field: 7]`? This seems more readable to me and consistent with other annotations.